data/NTH_2023/data.csv

```diff
@@ -10938,11 +10938,7 @@ NT_flora,Pycnospora lutescens,leaf_shape,leaflet,filiform,filiform,,mode
 NT_flora,Calopogonium mucunoides,leaf_shape,leaflet,linear,"linear,",,mode
 NT_flora,Cullen patens,leaf_shape,leaflet,elliptical lanceolate,elliptic- lanceolate,,mode
 NT_flora,Galactia megalophylla,leaf_shape,leaves or leaflets,elliptical widely_lanceolate ovate,"elliptic, widelylanceolate, ovate cordate,",,mode
-NT_flora,Senna obtusifolia,leaf_shape,leaflets,terete,terete,,mode
```

Comment:
I'm wondering why you've removed all the leaflet shapes? Didn't we agree they could be included as long as we specified as a context that it was a measurement of a leaflet. Or am I missing something else

Reply:
I removed just a few leaflet shapes in the Senna genus because the description of the leaflet only included the petiole shape - terete. It didn't include any lamina shape